pages/api/docs/getDpuHealth.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { getServerSession } from "next-auth/next";
+import { authOptions } from "../auth/[...nextauth]";
+import rudderStackEvents from "../events";
+import { getHealthStatusFromDB } from "../../../utils/db/healthStatus";
+
+const getDpuHealth = async (req: NextApiRequest, res: NextApiResponse) => {
+	const session = await getServerSession(req, res, authOptions);
+	if (!session || !session.user.id) {
```

Comment:
there might be a possibility that session does have use object. so we need to check for user object too.

Reply:
```suggestion
	if (!session || !session.user?.id) {
```